includes/controls/settings-header.php

```diff
@@ -0,0 +1,544 @@
+					'type' => Controls_Manager::SELECT,
+					'default' => 'tablet',
+					'options' => [
+						/* translators: %d: Breakpoint number. */
+						'mobile' => sprintf( __( 'Mobile (< %dpx)', 'hello-elementor' ), $breakpoints['md'] ),
+						/* translators: %d: Breakpoint number. */
+						'tablet' => sprintf( __( 'Tablet (< %dpx)', 'hello-elementor' ), $breakpoints['lg'] ),
+						'none' => __( 'None', 'hello-elementor' ),
+					],
+					'selector' => '.site-header',
+					'condition' => [
+						'hello_header_menu_layout!' => 'dropdown',
+					],
+				]
+			);
+
+			$this->add_control(
+				'hello_header_menu_color',
+				[
+					'label' => __( 'Color', 'hello-elementor' ),
+					'type' => Controls_Manager::COLOR,
+					'condition' => [
+						'hello_header_menu_display' => 'yes',
+					],
+					'selectors' => [
+						'.site-header ul.menu li a' => 'color: {{VALUE}};',
+					],
+				]
+			);
+
+			$this->add_control(
+				'hello_header_menu_toggle_color',
+				[
+					'label' => __( 'Toggle Color', 'hello-elementor' ),
+					'type' => Controls_Manager::COLOR,
+					'condition' => [
+						'hello_header_menu_display' => 'yes',
+					],
+					'selectors' => [
+						'.site-header .site-navigation-toggle i' => 'color: {{VALUE}};',
+					],
+				]
+			);
+
+			$this->add_group_control(
+				\Elementor\Group_Control_Typography::get_type(),
+				[
+					'name' => 'hello_header_menu_typography',
+					'label' => __( 'Typography', 'hello-elementor' ),
+					'condition' => [
+						'hello_header_menu_display' => 'yes',
+					],
+					'selector' => '.site-header .menu li',
+				]
+			);
+		}
+
+		$this->end_controls_section();
+	}
+
+	public function on_save( $data ) {
+		// Save chosen header menu to the WP settings.
+		if ( isset( $data['settings']['hello_header_menu'] ) ) {
+			$menu_id = $data['settings']['hello_header_menu'];
+			$locations = get_theme_mod( 'nav_menu_locations' );
+			$locations['menu-1'] = (int) $menu_id;
+			set_theme_mod( 'nav_menu_locations', $locations );
+		}
+	}
+
+	public function get_additional_tab_content() {
+		if ( ! defined( 'ELEMENTOR_PRO_VERSION' ) ) {
+			return wp_sprintf( '
+				<div class="elementor-nerd-box">
+					<img src="%4$s" class="elementor-nerd-box-icon">
+					<div class="elementor-nerd-box-message">
+						<p class="elementor-panel-heading-title elementor-nerd-box-title">%1$s</p>
+						<p>%2$s</p>
+					</div>
+					<a class="elementor-button elementor-button-default elementor-nerd-box-link" target="_blank" href="https://elementor.com/pro/?utm_source=panel-widgets&amp;utm_campaign=gopro&amp;utm_medium=wp-dash&amp;utm_term=helloelementor">%3$s</a>
+				</div>
+				',
+				__( 'Create a custom header with multiple options', 'hello-elementor' ),
+				__( 'Upgrade to Elementor Pro and enjoy free design and many more features', 'hello-elementor' ),
+				__( 'Go Pro', 'hello-elementor' ),
+				get_template_directory_uri() . '/assets/images/go-pro.svg'
+			);
+		} else {
+			return wp_sprintf( '
```

Comment:
?

Reply:
Why is `wp_sprintf()` used and not the standard `sprintf()`?

Reply:
Change all usage of `wp_sprintf` to the standard `sprintf()`

Reply:
Changed to `sprintf`